docs/src/using-deis/registering-a-user.md

```diff
@@ -8,7 +8,7 @@ To use Deis, you must first register a user on the [Controller][].
 Use `deis register` with the [Controller][] URL (supplied by your Deis administrator)
 to create a new account.  You will be logged in automatically.
 
-The domain you use here should match the one you set with `deisctl config platform set domain=`.
+The domain you use here should match the platform domain you selected when configuring the [Router][].
```

Comment:
Does https://github.com/deis/router/pull/67 change the recommendation to use a platform domain? Not blocking for this PR.